### PR TITLE
Introducing API requests

### PR DESCRIPTION
**Pull request by JuliCarracedo**
On this PR by @JuliCarracedo, The scores are saved inside the leaderboards API storage and retrieved from it to show.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+const formName = document.querySelector('input#name');
+const formScore = document.querySelector('input#score');
+const refresh = document.querySelector('button.btn');
+
+async function getStorage() {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3IGjFzzBEY0MX1krdUtm/scores/').then((response) => response.json()).then((json) => {
+    const list = json.result;
+    loadStored(list);
+  });
+}
```
Comment (oxenprogrammer) [inline src/index.js]: **Changes Required**
- [ ] The use of `arrow function` is preferred to the `function` keyword. Let us refactor this into an ES6 arrow function. You can read more [here](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why you should prefer ES6 arrow functions to the `function keyword`.
- [ ] Nice try using the `async await` syntactic sugar to wrap your asynchronous code. However, you're mixing it with the `then` keyword. Let us remove the `then` keyword and just use the pure `async await`.

File: src/postToAPI.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+async function postToAPI(score) {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/3IGjFzzBEY0MX1krdUtm/scores/', {
+    method: 'POST',
+    body: JSON.stringify(score),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((response) => response.json());
+}
```
Comment (oxenprogrammer) [inline src/postToAPI.js]: **Changes Required**
- [ ] The use of `arrow function` is preferred to the `function` keyword. Let us refactor this into an ES6 arrow function. You can read more [here](https://zendev.com/2018/10/01/javascript-arrow-functions-how-why-when.html) on why you should prefer ES6 arrow functions to the `function keyword`.
- [ ] Nice try using the `async await` syntactic sugar to wrap your asynchronous code. However, you're mixing it with the `then` keyword. Let us remove the `then` keyword and just use the pure `async await`.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @JuliCarracedo , 

Good job so far!
I can see you're really putting effort to become a world-class software developer :+1:
There are some issues that you still need to address in order to move forward to the next project.
Trust me you got this! :muscle:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Consider the optional changes nice to have features and only work on them when you have satisfied the required changes_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by carlos-ssh — APPROVED** [review]
Hi @JuliCarracedo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**